docs/flink.md

```diff
@@ -0,0 +1,52 @@
+# Flink
+
+If you're using Flink to process your event stream, you can use Tranquility's BeamSink to send data to
+Druid. The BeamSink can be added to any DataStream object. The BeamSink requires a BeamFactory to 
+propagate events. You will be overriding the makeBeam() function and within that
+using the DruidBeams builder's "buildBeam()" to build the beam. See the [DruidBeams documentation](druidbeams.md)
+for details about creating beams.
+
+It is recommended that you implement makeBeam as a lazy val so the Beam can be reused.
+
+For example:
+
+```scala
+
+class SimpleEventBeamFactory extends BeamFactory[SimpleEvent] {
+
+  lazy val makeBeam : Beam[SimpleEvent] = {
+
+    // Tranquility uses ZooKeeper (through Curator framework) for coordination.
+    val curator = CuratorFrameworkFactory.newClient(
+      "localhost:2181",
+      new BoundedExponentialBackoffRetry(100, 3000, 5)
+    )
+    curator.start()
+
+    val indexService = "druid/overlord" // Your overlord's druid.service, with slashes replaced by colons.
+    val firehosePattern = "druid:firehose:%s" // Make up a service pattern, include %s somewhere in it.
+    val discoveryPath = "/druid/discovery"     // Your overlord's druid.discovery.curator.path
+    val dataSource = "foo"
+    val dimensions = IndexedSeq("bar")
+    val aggregators = Seq(new LongSumAggregatorFactory("baz", "baz"))
+
+    DruidBeams
+      .builder((eventMap: SimpleEvent) => new DateTime())
```

Comment:
Using `new DateTime` for the timestamper is going to lead to mysterious drops (when tranquility thinks an event is ok and reports it sent, but druid silently drops it). I guess you copied this from the spark example, so we should fix both. I don't think that has to happen as part of this PR though.

Filed #93 to remember to do this.